code/game/gamemodes/objective.dm

```diff
@@ -339,7 +339,7 @@ datum/objective/hijack
 			return 0
 		var/area/shuttle = locate(/area/shuttle/escape/centcom)
 		for(var/mob/living/player in player_list)
-			if(istype(player, /mob/living/silicon) || istype(player, /mob/living/simple_animal) || player.mind.special_role && !player.mind.special_role == "Response Team")
+			if(istype(player, /mob/living/silicon) || istype(player, /mob/living/simple_animal) || player.mind.special_role)

```

Comment:
No idea, but it was; tested this specifically with a couple of people last night---had two traitors on the shuttle? Failed. Traitor+mindslaved? Failed.

Removed the ERT check->Two traitors? Success. Traitor+Mindslaved? Success.